src/components/data/kanban/kanban.stories.tsx

```diff
@@ -21,62 +19,37 @@ const meta: Meta<typeof Kanban> = {
 export default meta;
 type Story = StoryObj<typeof meta>;
 
+const generateComponentList = (count: number) => {
```

Comment:
I think it would be great to have cards with different content in the story, otherwise if you want to test the drag and drop functionality it is difficult to understand visually if it works as expected.

I also think that a story that tests the drag and drop and asserts what data is available in the callback would be helpful: it would make sure that there is enough information to pass back to the backend to update the state of the items.

Reply:
Noted, will make the story a bit more comprehensive